### DISCUSSION TITLE
Solve the LMI

### DISCUSSION THREAD

**Discussion by boubekeur27** [yalmip/YALMIP#1343] [General discussion]
Let the matrices be defined as follows 

A=\begin{bmatrix}
    -1 & -3 & 0 \\
    -3 & -4 & 1 \\0& 0 & -2 \\
\end{bmatrix}
and
E=\begin{bmatrix}
    1 & 2 & 3 \\
    1 & 2 & 3 \\
1 & 2 & 3 \\
\end{bmatrix}
and
B=\begin{bmatrix}
    0\\
    3\\ 
-2\\ 
\end{bmatrix} 
 and 
$F=\begin{bmatrix}
    sin(pi/3) & cos(pi/3) \\
    -cos(pi/3) &  sin(pi/3) \\
\end{bmatrix}$
and variables $X<0$ and $Y$ matrices
LMI is 

$$F \otimes A ^T(XE+BY^T) <0$$

$\otimes$ is Kronecker product .

**Comment by johanlofberg** [comment]
What is the question?

**Reply by boubekeur27** [reply]
solve  the LMI 
X and Y are variables

**Comment by johanlofberg** [comment]
To begin with, it is not an LMI as it is not symmetric, but once you have the correct form, there is nothing special here, you just implement it MATLAB as you've written it

https://yalmip.github.io/tutorial/basics/
https://yalmip.github.io/tutorial/semidefiniteprogramming/

**Reply by boubekeur27** [reply]
Thanks
